File: app/src/main/java/com/example/manajemenuser/app/MyApps.java
package com.example.manajemenuser.app;

import android.app.Application;

import com.facebook.stetho.Stetho;

public class MyApps extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Stetho.initializeWithDefaults(this);
        Stetho.initialize(
                Stetho.newInitializerBuilder(this)
                        .enableDumpapp(
                                Stetho.defaultDumperPluginsProvider(this))
                        .enableWebKitInspector(
                                Stetho.defaultInspectorModulesProvider(this))
                        .build());
    }
}

File: app/src/main/java/com/example/manajemenuser/app/MyApps.kt
package com.example.manajemenuser.app

import android.app.Application
import com.facebook.stetho.Stetho

class MyApps : Application() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
        Stetho.initialize(
            Stetho.newInitializerBuilder(this)
                .enableDumpapp(
                    Stetho.defaultDumperPluginsProvider(this)
                )
                .enableWebKitInspector(
                    Stetho.defaultInspectorModulesProvider(this)
                )
                .build()
        )
    }
}